### DISCUSSION TITLE
Installing Open WebUI with Bundled Ollama Support

### DISCUSSION THREAD

**Discussion by FrankOSITCP** [open-webui/open-webui#4723] [Ideas]
Hi everyone, 

Would it be possible to add bundled install, which will work with Intel GPUs as well? 
Current command in documentation: 
"docker run -d -p 3000:8080 --gpus=all -v ollama:/root/.ollama -v open-webui:/app/backend/data --name open-webui --restart always ghcr.io/open-webui/open-webui:ollama"

Above seems to support only Nvidia GPUs. 

When I run the command I will get the error: 
"(HTTP code 500) server error - failed to create task for container: failed to create shim task: OCI runtime create failed: runc create failed: unable to start container process: error during container init: error running hook #0: error running hook: exit status 1, stdout: , stderr: Auto-detected mode as 'legacy' nvidia-container-cli: initialization error: WSL environment detected but no adapters were found: unknown"

**Comment by silentoplayz** [comment]
Try this tutorial instead if you decide to follow along, but word of mouth is that Intel's custom IPEX-LLM build of Ollama is now _very_ out of date.

https://docs.openwebui.com/tutorial/ipex_llm
